web/src/pages/Route/Create.tsx

```diff
@@ -260,17 +260,10 @@ const Page: React.FC<Props> = (props) => {
     if (nextStep === 2) {
       if (step === 1) {
         form1.validateFields().then((value) => {
-          const { redirectOption, hosts } = value;
-          const filterHosts = hosts.filter(Boolean);
-          Promise.all([
-            redirectOption === 'forceHttps' && filterHosts.length !== 0
-              ? checkHostWithSSL(hosts)
-              : Promise.resolve(),
-            checkUniqueName(
-              value.name,
-              props.route.path.indexOf('edit') > 0 ? (props as any).match.params.rid : '',
-            ),
-          ]).then(() => {
+          checkUniqueName(
+            value.name,
+            props.route.path.indexOf('edit') > 0 ? (props as any).match.params.rid : '',
```

Comment:
type?

Reply:
This PR only deletes one function, and the rest of it is not expected to be changed.